### PR TITLE
Add docs for 0.13.0

### PR DESCRIPTION
**Pull request by vlad-perevezentsev**

### CODE REVIEW

File: 0.13.0/CONTRIBUTING.html [hunk 1]
```diff
@@ -0,0 +1,311 @@
+
+
+<!DOCTYPE html>
+<html class="writer-html5" lang="en" >
+<head>
+  <meta charset="utf-8" />
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  
+  <title>Contributing &mdash; numba-dppy 0.12.0 documentation</title>
```
Comment (PokhodenkoSA) [inline 0.13.0/CONTRIBUTING.html:10]: `numba-dppy 0.12.0` - old version.

Comment (PokhodenkoSA) [reply 0.13.0/CONTRIBUTING.html:10]: https://github.com/IntelPython/numba-dppy/blob/9c8dabf929368db96c3a2abf42072178b6cd9634/docs/conf.py#L26

Comment (vlad-perevezentsev) [reply 0.13.0/CONTRIBUTING.html:10]: @PokhodenkoSA Can I replace all lines with version 0.12 to 0.13 in this PR?

Comment (PokhodenkoSA) [reply 0.13.0/CONTRIBUTING.html:10]: Bad solution. It is better to fix root cause.

**Comment by PokhodenkoSA** [comment]
@vlad-perevezentsev from which sources you have generated this docs?

**Comment by vlad-perevezentsev** [comment]
@PokhodenkoSA Release0.13 branch

**Pull request merged**